test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -207,6 +207,7 @@ test
     await t.expect(pageTitle).contains('Verify with your phone');
     await t.expect(pageSubtitle).contains('Send a code via SMS to');
     await t.expect(pageSubtitle).contains('+1 XXX-XXX-2342');
+    await t.expect(challengePhonePageObject.elementExists('.okta-form-subtitle > span.no-translate.nowrap')).ok();
```

Comment:
the testcafe approach to adding coverage is because there is no unit test coverage for the two gen 2 files that were updated